packages/core-js/internals/uid.js

```diff
@@ -3,7 +3,7 @@ var uncurryThis = require('../internals/function-uncurry-this');
 
 var id = 0;
 var postfix = Math.random();
-var toString = uncurryThis(1.0.toString);
+var toString = uncurryThis(1.1.toString);
```

Comment:
This is a bug on your minificator side. I already added some workarounds for such kind of minificator bugs **to this line** (for example, `1..toString -> 1.0.toString)`, but I don't think we should encourage these bugs any further. I can accept this PR for compatibility with old versions of this tool, but only in case if this bug will be fixed in this tool itself.